src/Tribe/PUE/Checker.php

```diff
@@ -1173,5 +1204,55 @@ protected function get_site_domain() {
 				return strtolower( $site_url['host'] );
 			}
 		}
+
+		/**
+         * Returns the plugin file to use when checking for the licensed component or plugin.
+         *
+		 * @param string $plugin_file The component or plugin file in the `<dir>/<file>.php` format.
+		 *
+		 * @return string A component or plugins file in the `<dir>/<file>.php` format.
+		 */
+		protected function get_network_plugin_file( $plugin_file ) {
+		    $map = array(
```

Comment:
More funky whitespace :dancer: 

(Mix of tabs and spaces before `$map` and indentation for the next two lines is suspect)